lotus/settings.py

```diff
@@ -115,4 +115,5 @@ def __repr__(self) -> str:
 
 # set defaults
 settings = Settings()
-settings.configure(cascade_is_weight=0.5, cascade_num_calibration_quantiles=50, enable_cache=False)
+settings.configure(cascade_is_weight=0.5, cascade_num_calibration_quantiles=50, enable_cache=False) # Filter cascade settings
```

Comment:
Let's move the `enable_cache=False` outside the line of `# Filter cascade settings` since its unrelated.